AppEntry.js

```diff
@@ -2,6 +2,8 @@ import { BUILD_TYPE } from "@env"
 import { registerRootComponent } from "expo"
 import "TiFShared"
 
+console.log(BUILD_TYPE)
```

Comment:
Are we going to keep this console.log? XD
I know it works, but I hope this won't be intrusive

Reply:
No, but for some reason it seems that env values won't always reflect the most up to date value from the `.env` file unless we include this log...